web/src/components/storage/PartitionPage.tsx

```diff
@@ -625,6 +632,26 @@ function FilesystemSelect({
   );
 }
 
+type FilesystemLabelProps = {
+  id?: string;
+  value: string;
+  onChange: (v: string) => void;
+};
+
+function FilesystemLabel({ id, value, onChange }: FilesystemLabelProps): React.ReactNode {
+  const change = (v: string) => /^[\w-_.]*$/.test(v) && onChange(v);
+
+  return (
+    <TextInput
+      id={id}
+      aria-label={_("File system label")}
+      value={value}
+      onChange={(_, v) => change(v)}
+      className={sizingStyles.w_100OnLg}
```

Comment:
I don't think this is necessary. While we do use it occasionally, the goal is to keep its use to a minimum. In fact, we actually removed a few today.

From my testing, adding the w_100 here doesn't seem to make any difference.